app/serializers/current_user_serializer.rb

```diff
@@ -312,7 +313,15 @@ def has_topic_draft
     true
   end
 
+  def draft_count
+    Draft.stream({ user: object, all: true }).count
```

Comment:
This is doing too much work just for a count. 

```ruby
Draft.where(user: object).count
```

Should be enough and much efficient 👍